heartbeat/hbtest/hbtestutil.go

```diff
@@ -93,26 +95,33 @@ func TLSChecks(chainIndex, certIndex int, certificate *x509.Certificate) mapval.
 	})
 }
 
-// MonitorChecks creates a skima.Validator that represents the "monitor" field present
+// BaseChecks creates a skima.Validator that represents the "monitor" field present
 // in all heartbeat events.
-func MonitorChecks(host string, ip string, scheme string, status string) mapval.Validator {
+func BaseChecks(ip string, status string) mapval.Validator {
+
 	return mapval.MustCompile(mapval.Map{
 		"monitor": mapval.Map{
-			// TODO: This is only optional because, for some reason, TCP returns
-			// this value, but HTTP does not. We should fix this
-			"host":        mapval.Optional(mapval.IsEqual(host)),
 			"duration.us": mapval.IsDuration,
 			"ip":          ip,
-			"scheme":      scheme,
 			"status":      status,
 		},
 	})
 }
 
-// TCPBaseChecks checks the minimum TCP response, which is only issued
-// without further fields when the endpoint does not respond.
-func TCPBaseChecks(port uint16) mapval.Validator {
-	return mapval.MustCompile(mapval.Map{"tcp.port": port})
+// SimpleURLChecks returns a check for a simple URL
+// with only a scheme, host, and port
+func SimpleURLChecks(scheme string, host string, port uint16) mapval.Validator {
+
+	hostPort := host
+	if port != 0 {
+		hostPort = fmt.Sprintf("%s:%d", host, port)
+	}
+
+	u, _ := url.Parse(fmt.Sprintf("%s://%s", scheme, hostPort))
```

Comment:
error check skipped on purpose?

Reply:
Yes, my thinking is that it shouldn't happen with the limited input, but I don't mind making it a test failure.